src/components/cards/challenge/Challenge.tsx

```diff
@@ -2,7 +2,7 @@ import ArrowButton from "@/components/ui/button/Arrow";
 import { Community } from "@/types/community";
 import { Challenge } from "@/types/course";
 import Link from "next/link";
-import Badges from "./Badges";
+import Badges from "../../badges/Badges";
```

Comment:
If possible, could we rename the component "index", so that we don't have "badges/badges"